2025-03-28 00:46:50 | INFO     | __main__:main:114 - Starting ETL scheduler...
2025-03-28 00:46:50 | INFO     | __main__:main:120 - Running initial ETL job...
2025-03-28 00:46:50 | INFO     | __main__:run_etl:22 - Starting ETL process...
2025-03-28 00:46:50 | ERROR    | __main__:run_etl:92 - ETL process failed: no such table: job_metadata
2025-03-28 00:46:50 | WARNING  | __main__:run_etl_with_error_handling:106 - Attempt 1 failed. Retrying in 300 seconds...
